cypress/integration/apps/query_enhancements/dataset_selector.spec.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {
+  MiscUtils,
+  TestFixtureHandler,
+} from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+import { PATHS, SECONDARY_ENGINE } from '../../../utils/constants';
+
+const miscUtils = new MiscUtils(cy);
+const testFixtureHandler = new TestFixtureHandler(cy, PATHS.ENGINE);
+
+describe('dataset selector', { scrollBehavior: false }, () => {
+  describe('empty state', () => {
+    it('no index pattern', function () {
+      // Go to the Discover page
+      miscUtils.visitPage(
+        `app/data-explorer/discover#/?_g=(filters:!(),time:(from:'2015-09-19T13:31:44.000Z',to:'2015-09-24T01:31:44.000Z'))`
+      );
```

Comment:
I know you're migrating these, but thinking that these type of hardcoded visits are good opportunities for utils in the future.

Ex.
```
DiscoverPage.visit({
    timeRange: {
        from: '2015-09-19T13:31:44.000Z',
        to: '2015-09-24T01:31:44.000Z'
    }
});
```

Reply:
Seems we are still using the data from data-explorer. Will we do a follow-up to update the data? This is the one for query_enhancement https://github.com/opensearch-project/OpenSearch-Dashboards/pull/9006

Reply:
Sure, will do a follow up to update the data after 9006 is merged